libraries/AR_WPNav/AR_WPNav.cpp

```diff
@@ -291,8 +301,8 @@ bool AR_WPNav::use_pivot_steering(float yaw_error_cd)
         return true;
     }
 
-    // if within 10 degrees of the target heading, exit pivot steering
-    if (yaw_error < 10.0f) {
+    // if error is within _pivot_exit_angle of the target heading, exit pivot steering
+    if (yaw_error < _pivot_exit_angle) {
```

Comment:
I am not sure we want another parameter for this as _pivot_angle could be use here to control when to stop correcting. We could even use MIN(10.0f, _pivot_angle) for those that want fine tuning

Reply:
@khancyr This feature is actually supposed to help user's who want more precise pivot turns on a slower moving vehicle. So, a test case would be:` _pivot_angle = 60`, and `_pivot_exit_angle = 2`, so when yaw error to next WP is > 60, the vehicle would enter the pivot turn and keep pivoting till yaw error reaches 2 degrees. I don't think that will be possible without using the additional pram. The idea is to give the user the flexibility of deciding both: when to enter the pivot(`_pivot_angle`) and when to exit`(_pivot_exit_angle`). Both of them can be largely vehicle dependent. Please correct me if I am wrong! Thanks